.gitmodules

```diff
@@ -5,5 +5,4 @@
 [submodule "third_party/re2"]
 	path = third_party/re2
 	url = https://github.com/google/re2
-	ignore = dirty
-
+	ignore = dirty
```

Comment:
This seems like an unrelated change

Reply:
I think this is just a missing newline issue. Should be fixed